.azure-pipelines/run-test-scheduler-template.yml

```diff
@@ -47,12 +47,32 @@ steps:
     displayName: Trigger test
 
   - script: |
-      echo "Polling Test plan"
+      echo "Lock testbed"
+      echo "TestbedV2 is just online and might not be stable enough, for any issue, please send email to sonictestbedtools@microsoft.com"
+      echo "Runtime detailed progress at https://www.testbed-tools.org/scheduler/testplan/$TEST_PLAN_ID"
+      python ./.azure-pipelines/test_plan.py lock -i $(TEST_PLAN_ID) --timeout 300
```

Comment:
The CLI interface doesn't really make sense. It looks like to perform an action. But under the hood, it is polling the test plan to expect for some state.